### PR TITLE
added my revisions to cap_data ; added Makefile ; removed extra files

### PR DESCRIPTION
**Pull request by mccallm1**

### CODE REVIEW

File: src/image_capture/lifelong_object_learning/src/data_capture/capture_data.py [hunk 1]
```diff
@@ -19,38 +19,45 @@
 import sets
 import random
 
+# Added by Capstone
```
Comment (julianweisbord) [inline src/image_capture/lifelong_object_learning/src/data_capture/capture_data.py]: I don't think this is necessary

**Comment by julianweisbord** [comment]
Make sure to PEP8 everything, also you deleted: 'src/image_capture/prepare_data.py'

**Review by Rodrigmi — DISMISSED** [review]
Used wrong format for function headers. Refer to Julians function headers https://github.com/julianweisbord/Deep-Learning-For-Object-Recognition-on-a-Mobile-Robot/blob/master/src/learning/model.py

**Review by Rodrigmi — APPROVED** [review]
PEP8 Comments look good

**Pull request merged**